Sources/ProjectDescription/TestAction.swift

```diff
@@ -65,152 +65,87 @@ public struct TestAction: Equatable, Codable {
-            postActions: postActions,
-            diagnosticsOptions: diagnosticsOptions,
-            language: language.flatMap(SchemeLanguage.init(stringLiteral:)),
-            region: region
-        )
-    }
-
-    /// Initializes a new instance of a test action using test plans
-    /// - Parameters:
-    ///   - testPlans: List of test plans. The first in the list will be the default plan.
-    ///   - config: Configuration that should be used for building the test targets.
-    ///   - preActions: ist of actions to be executed before running the tests.
-    ///   - postActions: List of actions to be executed after running the tests.
-    public static func testPlans(_ testPlans: Path...,
-                                 config: PresetBuildConfiguration = .debug,
-                                 preActions: [ExecutionAction] = [],
-                                 postActions: [ExecutionAction] = []) -> Self
-    {
-        Self.testPlans(testPlans, config: config, preActions: preActions, postActions: postActions)
-    }
-
-    /// Initializes a new instance of a test action using test plans
-    /// - Parameters:
-    ///   - testPlans: List of test plans. The first in the list will be the default plan.
-    ///   - config: Configuration that should be used for building the test targets.
-    ///   - preActions: ist of actions to be executed before running the tests.
-    ///   - postActions: List of actions to be executed after running the tests.
-    public static func testPlans(_ testPlans: Path...,
-                                 configurationName: String,
-                                 preActions: [ExecutionAction] = [],
-                                 postActions: [ExecutionAction] = []) -> Self
-    {
-        Self.testPlans(testPlans, configurationName: configurationName, preActions: preActions, postActions: postActions)
-    }
-
-    /// Initializes a new instance of a test action using test plans
-    /// - Parameters:
-    ///   - testPlans: Array of test plans. The first in the array will be the default plan.
-    ///   - config: Configuration that should be used for building the test targets.
-    ///   - preActions: ist of actions to be executed before running the tests.
-    ///   - postActions: List of actions to be executed after running the tests.
+    ///   - language: The language to be used.
+    ///   - region: The region to be used.
+    /// - Returns: An initialized test action.
     public static func testPlans(_ testPlans: [Path],
-                                 config: PresetBuildConfiguration = .debug,
+                                 arguments: Arguments? = nil,
+                                 configuration: PresetBuildConfiguration = .debug,
+                                 coverage: Bool = false,
+                                 codeCoverageTargets: [TargetReference] = [],
+                                 expandVariableFromTarget: TargetReference? = nil,
```

Comment:
The previous implementation was correct, when using test plans only a subset of those options is available :(

There are two slightly different modes the scheme editor has

When using targets a lot of the options are exposed in the editor directly:

![targets](https://user-images.githubusercontent.com/11914919/131957756-4fbabde2-1141-47a0-ace2-9652ed1ef5df.png)



When using test plans however, it hides all the other options as they are defined within the individual test plans:

![testplans](https://user-images.githubusercontent.com/11914919/131957781-d598d45c-b190-4b1a-9a0b-581030d94abc.png)